lib/src/bloc/movies_bloc.dart

```diff
@@ -0,0 +1,19 @@
+import 'dart:async';
+import 'package:movies_app/src/models/item_model.dart';
+import '../resources/repository.dart';
+
+class MoviesBloc {
+  final _repository = Repository();
```

Comment:
Should we name this in a more meaningful way ? Like MoviesRepository ? or something like it?

Reply:
Done